ci_framework/roles/ci_setup/molecule/default/converge.yml

```diff
@@ -26,32 +26,70 @@
     - name: Default include
       ansible.builtin.import_role:
         name: ci_setup
+
     - name: Check directory creation
       block:
         - name: Stat directory
           register: dir_stat
           ansible.builtin.stat:
             path: "{{ ansible_user_dir }}/ci-framework-data/volumes"
+
         - name: Debug
           debug:
             msg: "{{ ansible_user_dir }}/ci-framework-data/volumes"
+
         - name: Validate directory state
           ansible.builtin.assert:
             that:
               - dir_stat.stat.isdir is defined
               - dir_stat.stat.isdir
+
+    - name: Verify openshift client
+      block:
+        - name: Get oc stat informaiton
+          register: oc_stat
+          ansible.builtin.stat:
+            path: "{{ ansible_user_dir }}/bin/oc"
+
+        - name: Verify the oc stat information
+          ansible.builtin.assert:
+            that:
+              - oc_stat.stat.exists is defined
+              - oc_stat.stat.exists | bool
+
+        - name: Verify openshift client version
+          register: oc_ver
+          environment:
+            PATH: "{{ cifmw_path }}"
+          ansible.builtin.command:
+            cmd: "oc version -o yaml"
+
+        - name: Debug - oc version output
+          ansible.builtin.debug:
+            msg: "{{ oc_ver.stdout }}"
+
+        - name: Compare the expected version
+          vars:
+            oc_ver_dict: "{{ oc_ver.stdout | from_yaml }}"
+            oc_min_ver: "{{ oc_ver_dict.releaseClientVersion | split('.') | first }}"
```

Comment:
that'd be `oc_maj_ver` I think, with the `|first` ?

Reply:
yes... you are correct. Fixing it :)